### PR TITLE
Fix TR5 laser head

### PR DESCRIPTION
**Pull request by Nemoel-Tomo**
TombEngine/Specific/savegame/schema/ten_savegame_generated.h
should **not** be included from this PR. (idk how to delete this file)

### CODE REVIEW

File: TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp [hunk 1]
```diff
@@ -1,670 +1,665 @@
...
+						guardian->target.x = target.x;
+						guardian->target.y = target.y;
+						guardian->target.z = target.z;
 					}
 					else
 					{
-						target.x = LaserHeadData.target.x;
-						target.y = LaserHeadData.target.y;
-						target.z = LaserHeadData.target.z;
+						target.x = guardian->target.x;
```
Comment (Lwmte) [inline TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp:230]: Why this condition even exists? `target` local variable is never used afterwards.

File: TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp [hunk 2]
```diff
@@ -1,670 +1,665 @@
...
+				{
+					int cosX = phd_cos(item->Pose.Orientation.x + GUARDIAN_ORIENT_OFFSET.x) * ANGLE(45.0f);
+
+					target.x = guardian->target.x = origin.x + cosX * phd_sin(item->Pose.Orientation.y);
+					target.y = guardian->target.y = origin.y + BLOCK(8) * phd_sin(GUARDIAN_ORIENT_OFFSET.x - item->Pose.Orientation.x);
+					target.z = guardian->target.z = origin.z + cosX * phd_cos(item->Pose.Orientation.y);
+				}
+				else
+				{
+					target.x = guardian->target.x;
```
Comment (Lwmte) [inline TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp:252]: Why this condition even exists? `target` local variable is never used afterwards.

Comment (Nemoel-Tomo) [reply TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp]: fixed

File: TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp (hunk 1 above)
Comment (Nemoel-Tomo) [reply TombEngine/Objects/TR5/Entity/tr5_laser_head.cpp]: fixed

**Pull request merged**